### PR TITLE
[bn] update validation for various correct translation values

### PR DESCRIPTION
**Pull request by ronydebnath**

### CODE REVIEW

File: src/bn/validation.php [hunk 1]
```diff
@@ -13,80 +13,80 @@
...
-    'alpha'                => ':attribute শুধুমাত্র অক্ষর থাকতে পারে.',
-    'alpha_dash'           => ':attribute শুধুমাত্র অক্ষর, সংখ্যা, এবং ড্যাশ থাকতে পারে.',
-    'alpha_num'            => ':attribute শুধুমাত্র বর্ণ ও সংখ্যা থাকতে পারে.',
-    'array'                => ':attribute একটি অ্যারে হতে হবে.',
-    'before'               => ':date :attribute সামনে একটি তারিখ হতে হবে.',
-    'before_or_equal'      => 'The :attribute must be a date before or equal to :date.',
+    'accepted'             => ':attribute গ্রহণ করা আবশ্যক।',
+    'active_url'           => 'এই :attribute একটি বৈধ URL নয়।',
+    'after'                => ':date :attribute পরে একটি তারিখ হতে হবে।',
+    'after_or_equal'       => 'The :attribute must be a date after or equal to :date।',
```
Comment (caouecs) [inline src/bn/validation.php]: Hello,
can you translate it ?

File: src/bn/validation.php [hunk 2]
```diff
@@ -13,80 +13,80 @@
...
+    'alpha'                => ':attribute শুধুমাত্র অক্ষর থাকতে পারে।',
+    'alpha_dash'           => ':attribute শুধুমাত্র অক্ষর, সংখ্যা, এবং ড্যাশ থাকতে পারে।',
+    'alpha_num'            => ':attribute শুধুমাত্র বর্ণ ও সংখ্যা থাকতে পারে।',
+    'array'                => ':attribute একটি অ্যারে হতে হবে।',
+    'before'               => ':date :attribute সামনে একটি তারিখ হতে হবে।',
+    'before_or_equal'      => 'The :attribute :date এর সাথে মিল অথবা এর আগের একটি তারিখ হতে হবে।',
```
Comment (caouecs) [inline src/bn/validation.php]: Hello,
"The" must be deleted, not ?

File: src/bn/validation.php [hunk 3]
```diff
@@ -13,80 +13,80 @@
...
+    'exists'               => 'নির্বাচিত :attribute টি অবৈধ।',
+    'file'                 => ':attribute একটি ফাইল হতে হবে।',
+    'filled'               => ':attribute স্থানটি পূরণ করতে হবে।',
+    'image'                => ':attribute একটি ইমেজ হতে হবে।',
+    'in'                   => 'নির্বাচিত :attribute টি অবৈধ।',
+    'in_array'             => ' :attribute উপাদানটি :other এ খুঁজে পাওয়া যায়নি।.',
+    'integer'              => ':attribute একটি পূর্ণসংখ্যা হতে হবে।',
+    'ip'                   => ':attribute একটি বৈধ  IP address হতে হবে।',
+    'ipv4'                 => 'The :attribute একটি বৈধ IPv4 address হতে হবে।',
+    'ipv6'                 => 'The :attribute একটি বৈধ IPv6 address হতে হবে।',
```
Comment (caouecs) [inline src/bn/validation.php]: Same problem with "the"

**Comment by caouecs** [comment]
Hello,
thank you, but your pull request must be from "master" branch

File: src/bn/validation.php (hunk 1 above)
Comment (ronydebnath) [inline src/bn/validation.php]: Sorry for my late response. I might have missed that one :) fixing it.

File: src/bn/validation.php (hunk 2 above)
Comment (ronydebnath) [inline src/bn/validation.php]: yes "The" has to be replaced with "টি". Adding that too.

File: src/bn/validation.php (hunk 3 above)
Comment (ronydebnath) [inline src/bn/validation.php]: Fixed.

**Comment by ronydebnath** [comment]
Sending another PR from master branch.

**Pull request closed without merging**